src/vs/workbench/services/host/electron-sandbox/nativeHostService.ts

```diff
@@ -193,8 +195,18 @@ class WorkbenchHostService extends Disposable implements IHostService {
 
 	//#region Screenshots
 
-	getScreenshot(): Promise<VSBuffer | undefined> {
-		return this.nativeHostService.getScreenshot();
+	getScreenshot(rect?: IRectangle): Promise<VSBuffer | undefined> {
+		return this.nativeHostService.getScreenshot(rect);
+	}
+
+	getElementData(offsetX: number, offsetY: number, token?: CancellationToken): Promise<IElementData | undefined> {
+		if (token) {
+			this._register(token.onCancellationRequested(() => {
```

Comment:
This leaks the disposable if `getElementData` is called repeatedly, individually dispose it after `getElementData` settles.